cms/plugin_rendering.py

```diff
@@ -481,6 +483,7 @@ def _render_page_placeholder(self, context, slot, page, editable=True, nodelist=
                 return nodelist.render(context)
             return ''
 
+        placeholder.is_inherit = inherit
```

Comment:
I'm -1 on this as we're trying to avoid these on-demand attributes.
We would have to check what is the best way to pass this information